src/llm/BUILD

```diff
@@ -73,7 +73,7 @@ mediapipe_proto_library(
 
 cc_library(
     name = "llmnoderesources",
-    hdrs = ["llmnoderesources.hpp",],
+    hdrs = ["llmnoderesources.hpp", "llm_executor.hpp",],

```

Comment:
Please add as separate library - faster compilation and less dependencies.

Reply:
I'm not sure it makes sense. LLM executor is tightly connected with the resources and is not really used outside of it. And at this point it's really not a lot of code.